dist/jszip-utils.js

```diff
@@ -70,6 +70,10 @@ JSZipUtils.getBinaryContent = function(path, callback) {
 
```

Comment:
Good PR.
I'm wondering if it's also possible to include here:

**xhr.withCredentials = true;**

This for most of the cases will do the job.

Cheers